examples/platform/silabs/SilabsDeviceDataProvider.cpp

```diff
@@ -389,7 +391,7 @@ CHIP_ERROR SilabsDeviceDataProvider::GetHardwareVersionString(char * buf, size_t
     if (err == CHIP_DEVICE_ERROR_CONFIG_NOT_FOUND)
     {
         VerifyOrReturnError(buf != nullptr, CHIP_ERROR_NO_MEMORY);
-        memcpy(buf, CHIP_DEVICE_CONFIG_DEFAULT_DEVICE_HARDWARE_VERSION_STRING, bufSize);
+        Platform::CopyString(buf, bufSize, CHIP_DEVICE_CONFIG_DEFAULT_DEVICE_HARDWARE_VERSION_STRING);
```

Comment:
Should probably have a strlen check here too.

Reply:
Yes, I forgot that one I must have pushed the commit while you were re-reviewing. It is there now.